orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -294,9 +294,9 @@ class TaskController {
     def evaluated = contextParameterProcessor.process(
       [expression: expression],
       [execution: execution],
-      false
+      true
```

Comment:
`true` here to enable helper functions (`stage`, `deployedServerGroups`, etc)

Reply:
wonder why this was false to begin with?

Reply:
@jeyrschabu Probably knows

Reply:
@danielpeach Any concern with this? (you added it w/ false in #1336 - not sure what the exact use case was there)

Reply:
nope, no memory of why though